src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  health = 100;
+
+  static alive = [];
+
+  static checkAlive() {
+    Animal.alive = Animal.alive.filter(beast => beast.health > 0);
+  }
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
```

Comment:
Herbivore has a method of hide, which changes the hidden property of the beast to the opposite value and helps to hide from carnivores.

Reply:
This method should change hidden property to the opposite value, not always to the true